src/frontend/styles/components/sidebar.pcss

```diff
@@ -117,6 +151,12 @@
     transition-property: background-color;
     transition-duration: 0.1s;
     @apply --squircle;
+    border: 2px solid transparent;
```

Comment:
this change breaks the Sidebar item height making it larger that in Figma. You can use `box-shadow` for outlining selected item instead

<img width="443" alt="image" src="https://user-images.githubusercontent.com/3684889/188509471-addaf9f4-454a-4cea-b617-96ac603a5f47.png">

Reply:
done